### ISSUE TITLE
Theming

### ISSUE THREAD

**Issue by torset1** [yeesoft/yii2-yee-cms#32]
Hello! 
Can help me?
I want use my theme. Say me please, where i can configure my theme that would not affect the folder yeesoft.
How do I use different themes in the best possible way,

**Comment by yeesoft** [comment]
@torset1 After installation you can freely update any code in frontend folder. This code will not be changed after updates from composer.

Use these files to change your theme:
`frontend\components\Theme`
`frontend\assets\ThemeAsset`

Best way to replace bootstrap css file is to update `frontend\config\main.php` to (for example):

```php
'components' => [
     'assetManager' => [
        'bundles' => [
            'yii\bootstrap\BootstrapAsset' => [
                'sourcePath' => '@app/themes/bootswatch/custom',
                'css' => ['bootstrap.css']
            ],
        ],
    ],
    'view' => [
        'theme' => [
            'class' => 'frontend\components\Theme',
            'theme' => 'custom', 
        ],
    ],
],
```

and remove `public $css = [
    'css/theme.css',
];` from `frontend\assets\ThemeAsset`.

You can use [bootswatch](https://bootswatch.com/help/) to create your theme. Maybe [yii2 theming guide](http://www.yiiframework.com/doc-2.0/guide-output-theming.html) will help you.

Feel free to change everything in frontend folder. During composer update only backend files will be updated.

**Comment by torset1** [comment]
Thank you for your answer and awesome cms.

**Comment by torset1** [comment]
Please tell me how do I add functionality in the backend to the upgrade he stayed?

**Comment by infario** [comment]
Yes. I want to update the backend layout as well. tries copy the layout/admin/main.php to backend view folder. but not working
